<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memur_giris</class>
 <widget class="QWidget" name="memur_giris">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>483</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="memur_ad">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>179</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="memur_sifre">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>179</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Kullancı Adı:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ŞiFRE:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="meur_giris">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: rgb(230, 115, 0);</string>
   </property>
   <property name="text">
    <string>Giriş</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#E67300;</string>
   </property>
   <property name="text">
    <string>MEMUR GİRİŞ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="giris">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>133</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="arka_plan">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>491</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>arka_plan</zorder>
  <zorder>memur_ad</zorder>
  <zorder>memur_sifre</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>meur_giris</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>giris</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
